src/screens/OrganizationFunds/OrganizationFunds.spec.tsx

```
@@ -67,51 +68,60 @@ const renderOrganizationFunds = (link: ApolloLink): RenderResult => {
+      };
+    });
   });
 
   afterEach(() => {
-    jest.clearAllMocks();
+    vi.clearAllMocks();
   });
 
   afterEach(() => {
     cleanup();
   });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)